src/main/java/org/json/JSONTokener.java

```diff
@@ -295,12 +295,9 @@ public String nextString(char quote) throws JSONException {
         StringBuilder sb = new StringBuilder();
         for (;;) {
             c = this.next();
-            switch (c) {
```

Comment:
Please revert all changes in this file.

Reply:
> Please revert all changes in this file. JSONTokener.java

All changes in the file are reverted, JSONTokener.java is no longer in the diff.

All requested changes are done.